moveit_core/collision_detection_bullet/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ install(FILES ../collision_detector_bullet_description.xml DESTINATION ${CATKIN_
 
 if(CATKIN_ENABLE_TESTING)
   find_package(moveit_resources REQUIRED)
-  include_directories(${moveit_resources_INCLUDE_DIRS})
+  include_directories(SYSTEM ${moveit_resources_INCLUDE_DIRS})
```

Comment:
This *is* our repository. We really shouldn't treat these as system includes.

Same in a number of other places.

Reply:
I agree for those.